api/initializer-db.go

```diff
@@ -0,0 +1,29 @@
+package api
```

Comment:
I believe this filename doesn't make sense with what is being done on its functions